bazel_tools/scala.bzl

```diff
@@ -49,6 +51,16 @@ common_scalacopts = [
     "-Xfatal-warnings",
     # catch missing string interpolators
     "-Xlint:missing-interpolator",
+    "-Xlint:by-name-right-associative",  # will never be by-name if used correctly
+    "-Xlint:constant",  # / 0
```

Comment:
What does "/ 0" mean?

Reply:
Divide by 0.

Reply:
I had guessed that much. It doesn't tell me what the linting rule does though. 😛

After reading the definition of `-Xlint:constant`, I guess this catches cases where we divide by `0` that only use constants? Still not sure though.